src/sentry/static/sentry/app/views/settings/organizationIntegrations/configureIntegration.tsx

```diff
@@ -49,6 +49,13 @@ class ConfigureIntegration extends AsyncView<Props, State> {
     ];
   }
 
+  componentDidMount() {
+    const {location} = this.props;
+    const value = location.query.tab === 'repos' ? 'repos' : 'codeMappings';
```

Comment:
@MeredithAnya we probably should flip the logic so that if we get a garbage tab value, we default to `repos`.

Reply:
good call